compiler/circle2circle/src/Circle2Circle.cpp

```diff
@@ -117,6 +117,41 @@ int entry(int argc, char **argv)
   arser.add_argument("input").nargs(1).type(arser::DataType::STR).help("Input circle model");
   arser.add_argument("output").nargs(1).type(arser::DataType::STR).help("Output circle model");
 
+  // sparsification argument
+  arser.add_argument("--sparsify_tensor")
```

Comment:
It would be better to make multiple options to one option.

Reply:
Q) is sparsify be possible for multiple tensors? then do we have to run multiple times of circle2circle for this?

Reply:
I'm not sure but thought of something like this;
```
--sparsify="TheTensor;NameForSparsify,0,d,10,0/Another;Tensor;Name,0,d,20,1"
```

Reply:
@seanshpark 
> Q) is sparsify be possible for multiple tensors? then do we have to run multiple times of circle2circle for this?

Yes. User passes specific tensor name that he/she wants to sparsify.

> It would be better to make multiple options to one option.

Actually, it is a little bit hard to make them one because the length of its array can be different for each tensor. To make it less verbose, I set a default value that is commonly used. Moreover, I will introduce a short option to `arser`.